frontend/src/components/EndGame/EndGame.component.jsx

```diff
@@ -37,10 +37,12 @@ const EndGame = () => {
   };
 
   const storeResult = async () => {
+    const token = window.localStorage.getItem('token')
     const response = await fetch("http://localhost:8000/game/update", {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
+        Authorization: `Bearer ${token}`,
```

Comment:
here I add Authorization to the post request so that the conditions for jwt.verify can be met